core/src/main/java/org/aerogear/mobile/core/metrics/MetricsPublisherListener.java

```diff
@@ -0,0 +1,11 @@
+package org.aerogear.mobile.core.metrics;
+
+
+import org.aerogear.mobile.core.http.HttpResponse;
+
+public interface MetricsPublisherListener {
+
+    void onPublishMetricsSuccess(HttpResponse response);
```

Comment:
It is a `MetricsPublisherListener` object, we already know it is about Metric publishing. No need to call the method `onPublshMetricsSuccess`. Better call it `onPublished`.

I don't like `response` to be passed here, agree with @danielpassos 

Something that helps the receiver know what has been processed should be passed, otherwise the listener will receive the event but won't know what the event was about.